Code/DataStructs/DatastructsException.h

```diff
@@ -19,7 +19,8 @@ class RDKIT_DATASTRUCTS_EXPORT DatastructsException : public std::exception {
   //! construct with an error message
   DatastructsException(const std::string &msg) : _msg(msg){};
   //! get the error message
-  const char *message() const { return _msg.c_str(); };
+  const char *what() const noexcept override { return _msg.c_str(); };
```

Comment:
I think I would prefer a base RDKitException that has these what and message exceptions.  ( I was doing this over the weekend )

The other benefit is that if we had a base RDKit Exception it would be easier to filter rdkit from other normal exceptions.

i.e.

```
DatastructsException : public RDKitException {
...
```

otherwise, this is awesome!

Reply:
After looking at the breadth of the exceptions, I don't think we need an RDKitException base class.  I'm pretty ecstatic with this.  Nice work.

Reply:
Thanks!